Assets/Scripts/UI/Menu's/MenuController.cs

```diff
@@ -3,19 +3,73 @@
 
 public sealed class MenuController : MonoBehaviour
 {
+    [Header("Main menu")]
     [SerializeField] private string mainScene;
     [SerializeField] private GameObject activeExplanationMenu;
     [SerializeField] private GameObject activeMainMenu;
 
+    [Header("Pause menu")] 
+    [SerializeField] private GameObject activePauseMenu;
+    [Space]
+    [SerializeField] private GameObject activePauseScreen;
+    [SerializeField] private GameObject activeOptionsScreen;
+    
+    private bool _setPauseMenu = true;
```

Comment:
```suggestion
    private bool _isNotPaused = true;
```
Het werkwoord SET gebruik je meer voor functies, voor bool is het meestal: is, has, was.
Waarom not er bij, is omdat de default value true is en de game begint niet paused of je moet allen ! omdraaien.